botathon/index.php

```diff
@@ -582,91 +582,104 @@ class="rd-parallax-layer section-top-75 section-md-top-150 section-lg-top-260">
 footer(false);
 ?>
 <script>
-	// var currentNav = "#nav-info"
-	// var currentAnchor = '#info'
-	var anchors = ['info','register','schedule','parts-list','teams','sponsors','contacts']
-	var curIndex = 0
-	var lastScrollTop = $(window).scrollTop()
+	let anchorSelectors = [$("#info"), $("#register"), $("#schedule"), $("#parts-list"), $("#teams"), $("#sponsors"), $("#contacts"),]
```

Comment:
We should have a querySelector or something that goes and grabs all of these dynamically, so we don't have to remember to update the JS every time.

Reply:
What do you mean update the JS every time? Do you mean the section headers might change?

Reply:
eg `document.querySelectorAll(".botathon-navigation li")` per meeting

Reply:
Resolved in 0a245a4